### PR TITLE
Adds slug to analytics screens - see #1063

### PR DESCRIPTION
**Pull request by orta**
We needed a slug identifier on these VCs - see #1063 

- [x] Artist
- [x] Artist Biography
- [x] Auction results
- [x] Fair artist
- [x] Fair personalized guide
- [x] internal_mobile_web

### CODE REVIEW

File: Artsy/App/ARAppDelegate+Analytics.m [hunk 1]
```diff
@@ -107,13 +107,6 @@ - (void)setupAnalytics
     hockey.disableCrashManager = YES;
 #endif
 
-    ARAnalyticsPropertiesBlock fairAndProfileIDBlock = ^NSDictionary*(ARFairGuideViewController *controller, NSArray *_) {
```
Comment (orta) [inline Artsy/App/ARAppDelegate+Analytics.m]: this was used once, no point in the extra abstraction

File: Artsy/App/ARAppDelegate+Analytics.m [hunk 2]
```diff
@@ -138,7 +131,13 @@ - (void)setupAnalytics
                         @{
                             ARAnalyticsEventName: ARAnalyticsFairGuideView,
                             ARAnalyticsSelectorName: NSStringFromSelector(@selector(viewDidAppear:)),
-                            ARAnalyticsProperties: fairAndProfileIDBlock
+                            ARAnalyticsProperties: ^NSDictionary*(ARFairGuideViewController *controller, NSArray *_) {
+                                return @{
+                                     @"profile_id" : controller.fair.organizer.profileID ?: @"",
+                                     @"fair_id" : controller.fair.fairID ?: @"",
+                                     @"slug": controller.fair.fairID
```
Comment (alloy) [inline Artsy/App/ARAppDelegate+Analytics.m]: These last two are the same value, but the latter is not guarded against.

Comment (orta) [inline Artsy/App/ARAppDelegate+Analytics.m]: :D - had force pushed that change as you wrote

**Comment by orta** [comment]
I want @anipetrov to give the Ok on this before merging.

**Comment by alloy** [comment]
I’m :+1: on the code itself, so @anipetrov may hit the merge button when she is satisfied imo.

(Be sure to also hit the “delete branch” button afterwards.)

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Error
    ------------- | ------------
        :no_entry_sign: | Developer Specific file shouldn't be changed
    

  

  
    &nbsp; | 1 Warning
    ------------- | ------------
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  
    &nbsp; | 1 Message
    ------------- | ------------
        :book: | Analytics dict changed, double check for ?: @""` on new entries
    

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Comment by orta** [comment]
whoah, wonder what's up with Danger?

**Comment by orta** [comment]
absorbed into #1145

**Pull request closed without merging**